src/main/java/org/folio/rest/impl/ItemStorageAPI.java

```diff
@@ -89,12 +54,13 @@ public void getItemStorageItems(
           PostgresClient postgresClient = StorageHelper.postgresClient(vertxContext, okapiHeaders);
 
           String[] fieldList = {"*"};
+          
```

Comment:
Can you configure your editor to trim trailing whitespace like these spaces in the empty line?
https://dev.folio.org/guides/developer-setup/#coding-style